module/observationSubmissions/helper.js

```diff
@@ -330,15 +341,21 @@ module.exports = class ObservationSubmissionsHelper {
                             completedDate: new Date()
                         }
                     );
+                    console.log("-------------------------Before calling pushObservationSubmissionForReporting------------");
                     await this.pushObservationSubmissionForReporting(submissionId);
+                    console.log("-------------------------After calling pushObservationSubmissionForReporting------------");
+                    console.log("--------------------------OBS WITH RUBRIC ISSUE DEGUG END rateSubmissionById()-------------------");
                     emailClient.pushMailToEmailService(emailRecipients,messageConstants.apiResponses.OBSERVATION_AUTO_RATING_SUCCESS+" - "+submissionId,JSON.stringify(resultingArray));
                     return resolve(messageConstants.apiResponses.OBSERVATION_RATING);
                 } else {
                     emailClient.pushMailToEmailService(emailRecipients,messageConstants.apiResponses.OBSERVATION_AUTO_RATING_FAILED+" - "+submissionId,JSON.stringify(resultingArray));
                     return resolve(messageConstants.apiResponses.OBSERVATION_RATING);
                 }
 
+                
+
             } catch (error) {
+                console.log("rateSubmissionById() catch block",error);
```

Comment:
@rakeshSgr this and error above log it in diff line and use JSON stringify

Reply:
updated